### PR TITLE
Update members 2024-2025

### PR DESCRIPTION
**Pull request by amarjanovicc**
This pull request updates the members list on the website for the 2024 year. The main changes include:

1. Adding new members and removing outdated ones
3. Updating descriptions for existing members
4. Image updates:
  -   New images for members added in JPG and WebP formats
  -   Optimized image sizes and quality
  -   Placeholder images set for members without photos

5. Impressum update:
  - Manually added new members and removed the outdated ones
  - Formatting and order have been verified to align with current standards

Reviewer: @Adam Marjanovic

### CODE REVIEW

File: static/clanovi/eva-petrovic – kopija.jpg
Comment (stakRedakcija) [inline static/clanovi/eva-petrovic – kopija.jpg:1]: Please rename

File: static/clanovi/rea-soboscan.jpg
Comment (stakRedakcija) [inline static/clanovi/rea-soboscan.jpg:1]: Please resize to 400 (on the larger side)

**Review by stakRedakcija — CHANGES_REQUESTED** [review]

File: static/clanovi/eva-petrovic – kopija.jpg
Comment (amarjanovicc) [reply static/clanovi/eva-petrovic – kopija.jpg:1]: Renamed

File: static/clanovi/rea-soboscan.jpg
Comment (amarjanovicc) [reply static/clanovi/rea-soboscan.jpg:1]: Resized to 400

Comment (stakRedakcija) [reply static/clanovi/rea-soboscan.jpg:1]: image
🤔

File: static/clanovi/morana-zuro.jpg
Comment (stakRedakcija) [inline static/clanovi/morana-zuro.jpg:1]: This one may need resizing too

File: static/clanovi/mateo-cuvalo.jpg
Comment (stakRedakcija) [inline static/clanovi/mateo-cuvalo.jpg:1]: Resize this one too, please

File: static/clanovi/matea-kucljak.jpg
Comment (stakRedakcija) [inline static/clanovi/matea-kucljak.jpg:1]: This one needs resizing

File: static/clanovi/marta-pamuklic.jpg
Comment (stakRedakcija) [inline static/clanovi/marta-pamuklic.jpg:1]: Resize please

File: static/clanovi/luka-arbutina.jpg
Comment (stakRedakcija) [inline static/clanovi/luka-arbutina.jpg:1]: This one needs resizing

File: static/clanovi/kaja-culjak.jpg
Comment (stakRedakcija) [inline static/clanovi/kaja-culjak.jpg:1]: Resize pls

File: static/clanovi/adam-marjanovic.jpg
Comment (stakRedakcija) [inline static/clanovi/adam-marjanovic.jpg:1]: Resize pls

**Review by stakRedakcija — APPROVED** [review]

**Pull request merged**